Codeinwp/otter-blocks
# feat: use editable actions for AI Toolbar

Closes https://github.com/Codeinwp/otter-internals/issues/215


### Summary


Added the toolbar actions as editable custom prompts.

Technical changes:
- new `wp_option` entry to save the custom actions
- I reworked the flow to hot-swap the pre-defined actions with the custom ones. 

### Screenshots 

![image](https://github.com/user-attachments/assets/e16f3c26-2701-4937-8f07-261d462039eb)

![image](https://github.com/user-attachments/assets/5dbac7ca-252a-4823-b2a3-90b8c68ee029)


https://github.com/user-attachments/assets/921e858e-c041-4ac8-8d42-dce80278199b


https://github.com/user-attachments/assets/df5e42cf-6d84-4ce2-a071-053498b5f222



----

### Test instructions


> [!NOTE]
> Make sure to have an OpenAI API Key

1. Insert a paragraph block
2. Check the new actions in AI Toolbar
3. Check if you can modify the actions 



---- 

### Checklist before the final review
